cet/shstk_alloc.c

```diff
@@ -80,6 +80,7 @@ int main(int argc, char *argv[])
 
 	if (!_get_ssp()) {
```

Comment:
Does it need to update here by using the new func `get_ssp`?

Reply:
Yes, it's better.

Thanks!